static/opensearch.xml

```diff
@@ -0,0 +1,7 @@
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
+	<ShortName>Charmhub</ShortName>
+	<Description>Charmhub - The Open Operator Collection</Description>
+	<InputEncoding>UTF-8</InputEncoding>
+	<Image width="16" height="16" type="image/x-icon">[https://assets.ubuntu.com/v1/5d4edefd-jaas-favicon.png]</Image>
```

Comment:
Does this image need to be 16x16 as its 256 at the moment. Also the square brackets here will fail?

Reply:
Square brackets are used in the example to show something to replace.

Reply:
Type should be "image/png" to. x-icon is for `.ico` images

Reply:
@anthonydillon I've replaced the image with 64x64 so that should make it look good on higher res screens too